doc/system_specific_setup.rst

```diff
@@ -1,225 +1,266 @@
-.. _Ubuntu14.03.3:
+.. _system_specific_setup:
 
 System-specific set-up
 ======================
+This chapter is split into two sections: :ref:`setup_user`
+describes the setup for a user of PSyclone. It includes all steps necessary
+to be able to use PSyclone. And while you could obviously do
+some development, none of the required tools for testing or
+documentation creation will be installed.
 
-This section provides system-specific information on how to set-up
-your system to use PSyclone.
+:ref:`dev_setup` describes the additional installation of
+all required tools to run tests, and create documentation.
 
-Ubuntu 14.04.3
---------------
+Both sections have detailled instructions for Ubuntu 16.04.2 and 
+OpenSUSE 42.2 - if you are working with a different Linux
+distribution some adjustments will be necessary.
 
-This guide has been tested with a vanilla installation of Ubuntu 14.04.3.
-
-.. _users:
+.. _setup_user:
 
 User set-up
-+++++++++++
+-----------
+
+This section provides system-specific information on how to
+set-up your system to use PSyclone for users.
+It has been tested with a vanilla installation of Ubuntu 16.04.2
+and OpenSUSE 42.2. You need a terminal window for entering the commands into.
+
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
+Most required dependencies are installed from the 
+Python Package Index (https://packaging.python.org/installing/)
+using the program pip ("PIP Installs Packages"). Besides ``pip``
+it is also recommended to install the graphviz package to be
+able to visualise dependency graphs. This is optional and the associated
+routine will silently return if the graphviz bindings are not
+installed.
 
-Get a terminal window. You can do this by pressing <ctrl><Alt><t>
-together, or click the top left "search" icon and type "terminal".
+.. _ubuntu_user:
 
-Install PSyclone
-^^^^^^^^^^^^^^^^
+Installing dependencies on Ubuntu
++++++++++++++++++++++++++++++++++
 
-Change directory to where you would like to place the code (where
-<PSYCLONEHOME> refers to where you would like to place the code):
+On Ubuntu ``pip`` and ``graphviz`` is installed using ``apt-get``. Remember
+that graphviz is optional.
```

Comment:
"and that you'll need to install the graphviz package in addition to the Python bindings."